docs/src/main/sphinx/installation/deployment.md

```diff
@@ -162,12 +162,18 @@ Because an `OutOfMemoryError` typically leaves the JVM in an
 inconsistent state, we write a heap dump, for debugging, and forcibly
 terminate the process when this occurs.
 
-The temporary directory used by the JVM must allow execution of code.
-Specifically, the mount must not have the `noexec` flag set. The default
-`/tmp` directory is mounted with this flag in some installations, which
-prevents Trino from starting. You can workaround this by overriding the
-temporary directory by adding `-Djava.io.tmpdir=/path/to/other/tmpdir` to the
-list of JVM options.
+(tmp-directory)=
+#### Temporary directory
+
+The temporary directory used by the JVM must allow execution of code, because
+Trino accesses and uses shared library binaries for purposes such as
+[](file-compression). 
+
+Specifically, the partition mount and directory must not have the `noexec` flag
+set. The default `/tmp` directory is mounted with this flag in some operating
+system installations, which prevents Trino from starting. You can workaround
+this by overriding the temporary directory by adding
+`-Djava.io.tmpdir=/path/to/other/tmpdir` to the list of JVM options.
```

Comment:
This should also be an option in jvm.config

Reply:
fixed